docker-compose.yml

```diff
@@ -3,7 +3,7 @@ x-frontend: &frontend
     VITE_ZONE: DEV
     VITE_USER_POOLS_ID: ca-central-1_t2HSZBHur
     VITE_USER_POOLS_WEB_CLIENT_ID: 3g6n2ha1loi4kp1jhaq359vrvb
-    VITE_BACKEND_URL: http://localhost:8080
+    VITE_BACKEND_URL: https://nr-silva-test-backend.apps.silver.devops.gov.bc.ca
```

Comment:
It was a mistake, I usually change it when the local testing is done, because my docker container cannot talk to the VPN so cant run the whole docker compose locally :(
Changed it :)